zebra-state/src/service.rs

```diff
@@ -754,7 +754,7 @@ impl StateService {
 
     /// Returns `true` if `queued_height` is near the final checkpoint.
     ///
-    /// The non-finalized block verifier needs access to UTXOs from finalized blocks
+    /// The checkpoint block verifier needs access to UTXOs from finalized blocks
     /// near the final checkpoint, so that it can verify blocks that spend those UTXOs.
```

Comment:
```suggestion
    /// The semantic block verifier needs access to UTXOs from checkpoint verified blocks
    /// near the final checkpoint, so that it can verify blocks that spend those UTXOs.
```